integration_tests/klog_test.go

```diff
@@ -135,14 +133,13 @@ func TestDestinationsWithDifferentFlags(t *testing.T) {
 			notExpectedInDir:    defaultNotExpectedInDirREs,
 		},
 		"with log dir and logtostderr": {
-			// Everything, EXCEPT the trace on fatal, goes to stderr. The -log_dir is
+			// Everything, including the trace on fatal, goes to stderr. The -log_dir is
 			// ignored, nothing goes to the log files in the log dir.
 
 			logdir: true,
 			flags:  []string{"-logtostderr=true", "-alsologtostderr=false", "-stderrthreshold=1000"},
 
-			expectedOnStderr:    res{infoLogRE, warningLogRE, errorLogRE, fatalLogRE},
-			notExpectedOnStderr: res{stackTraceRE},
+			expectedOnStderr: res{infoLogRE, warningLogRE, errorLogRE, fatalLogRE},
```

Comment:
Can you please change that `expectedOnStderr: res{infoLogRE, warningLogRE, errorLogRE, fatalLogRE, stackTraceRE}`, we now additionally expect the trace to show up in the output.